index.html

```diff
@@ -1232,6 +1232,18 @@ <h3>Special Ability</h3>
                   <td>ShldVamp</td>
                   <td>Can steal energy from player ship shields (unconfirmed)</td>
                 </tr>
+                <tr>
+                  <td><code>0x1000</code></td>
+                  <td>ELITE_?</td>
+                  <td>TeleBack</td>
```

Comment:
I assume therefore this is a name you have made up, not "official" in any way?

Reply:
The names "TeleBack" and "ShldReset" are what show up in the intel in the Science console.